src/shared/components/comment/comment-node.tsx

```diff
@@ -280,7 +280,7 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
                 isPostCreator={this.isPostCreator}
                 isModerator={creator_is_moderator}
                 isAdmin={creator_is_admin}
-                isBot={cv.creator.bot_account}
+                person={cv.creator}
```

Comment:
Pry better to call this creator, and remove the `targetPersonId` prop also.

Reply:
Done